tests/science/vic_parameters_42.txt

```diff
@@ -0,0 +1,276 @@
+ # @section DESCRIPTION
```

Comment:
I like this format better than `global_param.classic.5.0_xxxx.txt` because it has explanation and units of the variables. The user can use/change this file more easily.